src/deepsparse/transformers/pipelines/masking.py

```diff
@@ -0,0 +1,336 @@
+                    return args[0]
+                return self.input_schema(inputs=args[0])
+            else:
+                return self.input_schema(inputs=args)
+
+        return self.input_schema(**kwargs)
+
+    def process_inputs(
+            self,
+            inputs: MaskingInput,
+    ) -> Tuple[List[numpy.ndarray], Dict[str, Any]]:
+        """
+        :param inputs: inputs to the pipeline. Must be the type of the
+            MaskingInput
+        :return: inputs of this model processed into a list of numpy arrays that
+            can be directly passed into the forward pass of the pipeline engine
+            and dictionary containing offset mappings and special tokens mask to
+            be used during postprocessing
+        """
+        if inputs.is_split_into_words and self.engine.batch_size != 1:
+            raise ValueError("is_split_into_words=True only supported for batch size 1")
+
+        tokens = self.tokenizer(
+            inputs.inputs,
+            return_tensors="np",
+            truncation=TruncationStrategy.LONGEST_FIRST.value,
+            padding=PaddingStrategy.MAX_LENGTH.value,
+            return_special_tokens_mask=True,
+            return_offsets_mapping=self.tokenizer.is_fast,
+            is_split_into_words=inputs.is_split_into_words,
+        )
+
+        offset_mapping = (
+            tokens.pop("offset_mapping")
+            if self.tokenizer.is_fast
+            else [None] * len(inputs.inputs)
+        )
+        special_tokens_mask = tokens.pop("special_tokens_mask")
+
+        word_start_mask = None
+        if inputs.is_split_into_words:
+            # create mask for word in the split words where values are True
+            # if they are the start of a tokenized word
+            word_start_mask = []
+            word_ids = tokens.word_ids(batch_index=0)
+            previous_id = None
+            for word_id in word_ids:
+                if word_id is None:
+                    continue
+                if word_id != previous_id:
+                    word_start_mask.append(True)
+                    previous_id = word_id
+                else:
+                    word_start_mask.append(False)
+
+        postprocessing_kwargs = dict(
+            inputs=inputs,
+            tokens=tokens,
+            offset_mapping=offset_mapping,
+            special_tokens_mask=special_tokens_mask,
+            word_start_mask=word_start_mask,
+        )
+
+        return self.tokens_to_engine_input(tokens), postprocessing_kwargs
+
+    def process_engine_outputs(
+            self,
+            engine_outputs: List[numpy.ndarray],
+            **kwargs,
+    ) -> BaseModel:
+        """
+        :param engine_outputs: list of numpy arrays that are the output of the engine
+            forward pass
+        :return: outputs of engine post-processed into an object in the `output_schema`
+            format of this pipeline
+        """
+
+        topK = 10
```

Comment:
perhaps set it as an argument and not hard code it?

Reply:
Have added this as argument @dbogunowicz